spec/features/dog_filter_spec.rb

```diff
@@ -9,9 +9,9 @@
   end
 
   let!(:active_user) { create(:user) }
-  let!(:primary_lab) { create(:dog, :primary_lab, name: "Zeke").name.titleize }
-  let!(:secondary_golden) { create(:dog, :secondary_golden, name: "Abby").name.titleize }
-  let!(:secondary_westie) { create(:dog, :secondary_westie, name: "Nairobi").name.titleize }
+  let!(:primary_lab) { create(:dog, :primary_lab, :active_dog, name: "Zeke").name.titleize }
```

Comment:
what is going on here?

Reply:
added an `:active_dog` trait so that the dogs that get created for this test get assigned one of the ACTIVE_STATUSES.   By adjusting the links to the Dog Manager in this PR, Dog Manager will default to showing only dogs that have one of the active statuses.

Reply:
I get that part but why the `name.titleize` ?

Reply:
ah nvm, that was there already, carry on!

Reply:
oops, yes I did use the variable names, but not consistently!